packages/mediapipe-core/lib/src/mediapipe_common_bindings.dart

```diff
@@ -0,0 +1,332 @@
+// AUTO GENERATED FILE, DO NOT EDIT.
```

Comment:
Code generated from a file with copyright X falls under copyright X, include the copyright header from the C source file.

Consider moving this into lib/src/third_party